vendor/llm2vec_monGARS/llm2vec/dataset/E5Data.py

```
@@ -0,0 +1,156 @@
+
+        self.data = [all_samples[idx] for idx in final_idx_order]
+        logger.info(f"Loaded {len(self.data)} samples.")
+
+    def __getitem__(self, index):
+        sample = self.data[index]
+        if self.split == "train":
+            return TrainSample(
+                texts=[sample.query, sample.positive, sample.negative], label=1.0
+            )
+        elif self.split == "validation":
+            assert False, "E5Data does not have a validation split."
```

Ruff (0.13.3)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)